src/app/zap-templates/partials/header.zapt

```diff
@@ -15,4 +15,6 @@
  *    limitations under the License.
  */
 
+{{#unless excludeZapComment}}
```

Comment:
The PR summary does not explain why this is done: don't we want to know which files are not to be edited by humans?

Reply:
These are public SDK headers on Darwin.  They should not mention ZAP.   See the "Review notes" bit in https://github.com/project-chip/connectedhomeip/pull/22551 (which this is re-landing).